libs/sdk-backend-tiger/src/utils/downloadFile.ts

```diff
@@ -10,3 +12,11 @@ export function downloadFile(fileName: string, data: any) {
     URL.revokeObjectURL(link.href);
     document.body.removeChild(link);
 }
+
+export const parseNameFromContentDisposition = (response: AxiosResponse) => {
```

Comment:
This method is identical to method in [libs/api-client-bear/src/utils/export.ts]

Reply:
I had it originally shared but it is a bit different. One is accepting `AxiosResponse` the second fetch's `Response`. I did not want to unpack header value outside of it as I would have to do it three times, if I remember correctly.

However, the actual parsing with regexes could be shared. What would be the correct place for such a util as I require to use it in both Tiger and Panther API clients?

Reply:
On the other hand, possibly it made sense that regex parsing is duplicated as the response is served by two different service (export service for Bear and Tiger) and we could keep these separate if one of them would change. This is however highly theoretical. If there is place where it can be put to have it shared, I will gladly change it.